### PR TITLE
Add support for netAnim

### PR DESCRIPTION
**Pull request by patel999jay**
Hello,

Please review code and merge the same.
Feature added : netAnim support added, confirm and tested.
NetAnim-WOSS-Tracing
WOSS-Random-Disc-PacketAnalysis
WOSS-Grid_Simulation_4Nodes
WOSS-PacketsTracing
WOSS-List_1stTransmission
WOSS-NetAnim-ListNode

### CODE REVIEW

File: wscript [hunk 1]
```diff
@@ -136,7 +136,7 @@ def build(bld):
     if 'woss-ns3' in bld.env['MODULES_NOT_BUILT']:
         return
 
-    module = bld.create_ns3_module('woss-ns3', ['network', 'energy', 'mobility', 'uan'])
+    module = bld.create_ns3_module('woss-ns3', ['netanim','network', 'energy', 'mobility', 'uan'])
```
Comment (MetalKnight) [inline wscript]: please add a space after the comma between netanim and network

`    module = bld.create_ns3_module('woss-ns3', ['netanim', 'network', 'energy', 'mobility', 'uan'])`

File: examples/wscript [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## -*- Mode: python; py-indent-offset: 4; indent-tabs-mode: nil; coding: utf-8; -*-
 
 def build(bld):
-    obj = bld.create_ns3_program('woss-aloha-example', ['internet', 'mobility', 'stats', 'applications', 'uan', 'woss-ns3'])
+    obj = bld.create_ns3_program('woss-aloha-example', ['netanim', 'internet', 'mobility', 'stats', 'applications', 'uan', 'woss-ns3'])
```
Comment (MetalKnight) [inline examples/wscript]: the example is not using netAnim explicitly. 
Is this really required?

Comment (patel999jay) [reply examples/wscript]: In general, it is good to have netanim support for post-processing purposes. I had modified the example as well to build support for netanim to test it, but as you said this example is not using netAnim explicitly so I only push changes related to wscript to pre-build netAnim support for any user by default in the example. 
It is solely up to user to use netanim for their simulation or not.

Comment (MetalKnight) [reply examples/wscript]: Ok good, thanks! 
I will merge this tomorrow.

**Comment by MetalKnight** [comment]
closes #26

**Pull request merged**